data/mods/Magiclysm/Spells/attunements/Crusader.json

```diff
@@ -120,14 +120,20 @@
     "material": [ "concentrated_mana" ],
     "weapon_category": [ "LONG_SWORDS", "GREAT_SWORDS" ],
     "techniques": [ "RSTRIKE", "WBLOCK_2", "BRUTAL", "RAPID" ],
-    "flags": [ "UNBREAKABLE_MELEE", "NONCONDUCTIVE", "NO_REPAIR", "NO_SALVAGE", "MAGIC_FOCUS", "TRADER_AVOID", "SHEATH_SWORD" ],
+    "flags": [
+      "UNBREAKABLE_MELEE",
+      "NONCONDUCTIVE",
+      "NO_REPAIR",
+      "NO_SALVAGE",
+      "MAGIC_FOCUS",
+      "TRADER_AVOID",
+      "SHEATH_SWORD",
+      "RELIC_PINK"
+    ],
     "to_hit": { "grip": "weapon", "length": "long", "surface": "line", "balance": "good" },
     "weight": "1067 g",
     "volume": "2750 ml",
     "longest_side": "120cm",
-    "relic_data": {
```

Comment:
It looks like pure damage was not added to the damage definitions for the items in this file after it was removed from the relic data

Reply:
Yes, it was intentional decision, they already have light damage bonus defined, defining two damage bonuses is a mistake